glusterd2/commands/volumes/volume-option.go

```diff
@@ -43,7 +43,7 @@ func optionSetValidate(c transaction.TxnCtx) error {
 		return err
 	}
 
-	if err := validateXlatorOptions(req.Options, &volinfo); err != nil {
+	if err := validateXlatorOptions(options, &volinfo); err != nil {
 		return fmt.Errorf("Validation failed for volume option:: %s", err.Error())
```

Comment:
TRIVIAL :  @prashanthpai @Madhu-1 Should this statement start with lower case as well? Not part of changes made in this PR though.

Reply:
Sure